packages/dashboard/src/components/elements/button/button.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import classNames from 'classnames';
+import { Button as CButton } from '@deriv/components';
+import { TReactChildren } from 'Types'
+
+const Button: React.FC<TButton> = ({ className, children, onClick, ...props }) => {
```

Comment:
Instead of this component, can we just override css for `.dc-btn` in dashboard package?
There are components (e.g. FormSubmitButton) in `components`  package that uses `Button` from `components` package. If we use `FormSubmitButton` component in `dashboard` then we will still have the default style.

Would it be better to override `.dc-btn` style in order to change all buttons style in `dashboard`?

Reply:
thanks for the suggestion bala, yes, that would be better

Reply:
which one do you think better this approach, or i just add specific new type of button in component package?

Reply:
I think it's better to override just the styles for buttons in dashboard package instead of creating a wrapper component for button.

Reply:
One thing to note is that it makes the component less maintainable. Developers will need to remember that whenever they change a component (e.g. add/remove a class) in `@deriv/components` (in this case `Button`) that it will affect the usage in `@deriv/dashboard`. However, we can probably get away with this in Deriv.app considering it's built as one big app.